### PR TITLE
Replacing PyMC3 plots w/ Arviz plots & sigma Param change [Part 5]

### PR DESCRIPTION
**Pull request by CloudChaoszero**
# Description
The following is a large PR breakdown of PR #16.

Replace PyMC3 dependent plots with arviz plots in case studies & examples.
* The issue is a 3-part step for the following:
  * Initiative via Issue https://github.com/pymc-devs/pymc3/issues/4371 (Leave open)

Replace parameter `sd` with `sigma` (e.g. some examples have `pm.Normal(...sd=...)`
* Closes issue #2

### CODE REVIEW

File: examples/ode_models/ODE_with_manual_gradients.ipynb [hunk 1]
```diff
@@ -6,6 +6,9 @@
    "metadata": {},
```
Comment (twiecki) [inline examples/ode_models/ODE_with_manual_gradients.ipynb:6]: <p>This plot isn't displaying for some reason.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (twiecki) [inline examples/ode_models/ODE_with_manual_gradients.ipynb:6]: <p>Missing output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (twiecki) [inline examples/ode_models/ODE_with_manual_gradients.ipynb:6]: <p>Missing output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (CloudChaoszero) [reply examples/ode_models/ODE_with_manual_gradients.ipynb]: <p>Yeah, I realized I didn't properly re-run the notebook. In order to do so, I resolved issues like:</p><ul><li>Specifying <code>core</code>  param on <code>pm.sample</code></li><li>Update submodule ref (e.g From theano.Op to theano.tensor.Op)</li></ul><p> </p>

**Pull request merged**